subsys/bluetooth/mesh/shell/shell_ctrl_cli.c

```diff
@@ -190,7 +208,14 @@ static void prop_print(const struct shell *shell, int err, struct sensor_value *
 
 static int cmd_prop_get(const struct shell *shell, size_t argc, char *argv[])
 {
-	enum bt_mesh_light_ctrl_prop id = (enum bt_mesh_light_ctrl_prop)strtol(argv[1], NULL, 0);
+	int err = 0;
+	enum bt_mesh_light_ctrl_prop id =
+		(enum bt_mesh_light_ctrl_prop)shell_strtol(argv[1], 10, &err);
```

Comment:
Scope creep so feel free to ignore, but: We should really check the values when casting to enum types. As it is now it can cause undefined behavior.

Reply:
Agreed, but I'd prefer to do that as a separate issue.